src/pages/dashboard/ArrivalByTimeChart/ArrivalByTimeChart.tsx

```diff
@@ -51,7 +51,7 @@ export default function ArrivalByTimeChart({
         .map((group) => (
           <div key={group[0].name}>
             <h3 className="title">{group[0].name}</h3>
-            <ResponsiveContainer debounce={1000} width="100%" height={300}>
+            <ResponsiveContainer debounce={1000} width="100%" minWidth={300} height={300}>
```

Comment:
what's if the screen is narrower, or wider? I'm just not sure `minWidth` is the right solution for all devices

Reply:
using negative left margin on the component seems to have a similar effect, what do you think?
![image](https://github.com/hasadna/open-bus-map-search/assets/54629307/ad060be3-0d0c-44d0-ad08-675c74da65ca)

Reply:
negative margin is also a bad practice
I think [this thread](https://stackoverflow.com/questions/50891591/recharts-responsive-container-does-not-resize-correctly-in-flexbox) worth investigation - it seems like a very similar problem

Reply:
I've looked into it and after experimenting the solution that worked for me is updating the package and setting a right margin of 35, but I'm still not sure if that's the best solution

Reply:
That's far better! Thank you for doing this research